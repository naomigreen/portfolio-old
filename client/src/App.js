import React from 'react';
import styled from 'styled-components';
import Routes from './containers/routes/Routes';

function App() {
  const video = 'https://website-background.s3.eu-west-2.amazonaws.com/bg-compressed.mp4';
  return (
    <AppWrapper>
      <Video autoPlay="autoplay" loop="loop" muted>
        <source src={video} type="video/mp4" />
      </Video>
      <Content>
        <Routes />
      </Content>
    </AppWrapper>
  )
}

const AppWrapper = styled.div`
  right: 0;
	bottom: 0;
  width: 100%;
  height: 100%;
`;

const Video = styled.video`
  width: 100%;
  height: 100%;
  position: fixed;
  object-fit: cover;
`;

const Content = styled.div`
  position: relative;
  top: 0;
  width: 100%;
  height: 100%;
  display: flex;
  justify-content: center;
  align-items: center;
  z-index: 2;
`;

export default App;