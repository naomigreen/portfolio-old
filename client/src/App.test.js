import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

describe('App tests', () => {
	const { asFragment } = render(<App />);

	it('should render App component', () => {
		expect(asFragment()).toMatchSnapshot();
	});
});
