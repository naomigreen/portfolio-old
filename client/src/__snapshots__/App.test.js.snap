// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App tests should render App component 1`] = `
<div>
  <div
    class="sc-fzoXzr jUsjNB"
  >
    <video
      autoplay=""
      class="sc-fzpjYC crYJYo"
      loop=""
    >
      <source
        src="https://website-background.s3.eu-west-2.amazonaws.com/bg-compressed.mp4"
        type="video/mp4"
      />
    </video>
    <div
      class="sc-fznxsB cQMWzy"
    >
      <div
        class="sc-fzqBZW BQyur"
      >
        <h2
          class="sc-fzoLag fiMHy"
        >
          Codes Green Ltd
        </h2>
        <div
          class="sc-fzqNJr dbelry"
        >
          <a
            aria-current="page"
            class="sc-fzoyAV dMQIdv active-link"
            href="/home"
          >
            Home
          </a>
          <a
            class="sc-fzoyAV dMQIdv"
            href="/work"
          >
            Work
          </a>
          <a
            class="sc-fzoyAV dMQIdv"
            href="/demo"
          >
            Demo
          </a>
          <a
            class="sc-fzoyAV dMQIdv"
            href="/contact"
          >
            Contact
          </a>
        </div>
        <div
          class="sc-fznKkj bJZima"
        >
          <img
            alt=""
            class="sc-AxjAm fqTcMn"
            radius="100%"
            src="https://naomi-assets.s3.eu-west-2.amazonaws.com/me.jpg"
            width="150px"
          />
          <div
            class="sc-fznyAO gOGuAv"
          >
            <p>
              I’m Naomi Prescod-Green, a Full Stack JavaScript Contractor and founder of Codes Green Ltd. 
              <br />
               
I have a Computer Science background and specialise in React, Vanilla and Node JavaScript. 
              <br />
               
I enjoy working on complex projects that test my skills and teach me new approaches to software 
development.
            </p>
          </div>
        </div>
        <footer
          class="sc-fzplWN hYgkwo"
        >
          <p>
            © Codes Green Ltd 
          </p>
        </footer>
      </div>
    </div>
  </div>
</div>
`;
